### PR TITLE
Group likes button on main group page

### LINKED ISSUE #552: Add "Like" button to Groups' main page
The "like" button for groups only appears in 2 areas:

1- In the list of groups a user is a member of which is displayed on the right hand side of a user's profile page.
2- In the list of Featured groups if a user clicks on the Groups Tab

Request to Add "Like" button to Groups' main page and/or by the Group's statistics when groups are viewed in a list

### PR DESCRIPTION
**Pull request by piet0024**
A 'Like this group' button now appears on the group main page.

Closes #552

### CODE REVIEW

File: mod/gc_group_layout/views/default/groups/profile/summary.php [hunk 1]
```diff
@@ -305,6 +305,34 @@
...
+			                     'name' => 'unlike',
+			                     'href' => elgg_add_action_tokens_to_url("/action/likes/delete?guid={$group->guid}"),
+			                     'text' => '<i class="fa fa-thumbs-up fa-lg icon-sel"></i><span class="wb-inv">'. elgg_echo('likes:remove').'</span>',
+			                     'title' => elgg_echo('likes:remove') . ' ' .elgg_echo('group'),
+			                     'item_class' => $hasLiked ? 'pad-rght-xs' : 'hidden',
+			                     'priority' => 998,
+		                      );
+                            }else{
+                               $options = array(
+			                     'name' => 'likes',
```
Comment (Phanoix) [inline mod/gc_group_layout/views/default/groups/profile/summary.php]: Shouldn't it be called 'like' if the opposite is 'unlike'?

Comment (piet0024) [inline mod/gc_group_layout/views/default/groups/profile/summary.php]: or perhaps it could be 'unlikes' 🤔

Comment (piet0024) [inline mod/gc_group_layout/views/default/groups/profile/summary.php]: These links do not need 'names' so i am going to remove them anyways

Comment (Phanoix) [inline mod/gc_group_layout/views/default/groups/profile/summary.php]: That works too, I guess

**Pull request merged**